buildSrc/private/src/main/kotlin/androidx/build/AndroidXComposeMultiplatformExtensionImpl.kt

```diff
@@ -216,88 +227,148 @@ fun enableOELPublishing(project: Project) {
     }
 }
 
-// FIXME: reflection access! Some API in Kotlin is needed
-@Suppress("unchecked_cast")
-private val KotlinTarget.kotlinComponents: Iterable<KotlinTargetComponent>
-    get() = javaClass.kotlin.memberProperties
-        .single { it.name == "kotlinComponents" }
-        .get(this) as Iterable<KotlinTargetComponent>
 
+/**
+ * Usage that should be added to rootSoftwareComponent to represent android-specific variants
+ * It will be serialized to *.module in "variants" collection.
+ */
+private class FakeAndroidUsage(
```

Comment:
```suggestion
private class CustomAndroidUsage(
```

This is a valid functionality, not fake :)

Reply:
done